coral-component-colorinput/src/templates/base.html

```diff
@@ -21,14 +21,15 @@
 <input handle="input" is="coral-textfield" class="_coral-ColorInput-input" type="text" value="" role="combobox" aria-autocomplete="none" aria-haspopup="dialog" aria-expanded="false" aria-controls="{{uid}}" />
 <div class="_coral-ColorInput-buttonWrapper" handle="buttonWrapper" role="presentation">
   <button handle="colorPreview" is="coral-button" variant="_custom" class="_coral-FieldButton _coral-ColorInput-button _coral-ColorInput-preview" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="{{uid}}"></button>
+  <label handle="colorPreviewLabel" type="label"></label>
```

Comment:
Simply including the color value as a block element breaks the layouts of both the ColorInput and ColorInputSwatches. It may be better to use a CoralTooltip to display the value on hover or focus.

```suggestion
  <button handle="colorPreview" is="coral-button" variant="_custom" class="_coral-FieldButton _coral-ColorInput-button _coral-ColorInput-preview" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="{{uid}}" id="{{uid}}-colorPreview" aria-describedby="{{uid}}-tooltip"></button>
  <coral-tooltip handle="colorPreviewLabel" class="_coral-ColorInput-preview-tooltip" id="{{uid}}-tooltip" target="#{{uid}}-colorPreview" placement="top"></coral-tooltip>

```